nilearn/_utils/param_validation.py

```diff
@@ -63,3 +63,31 @@ def check_threshold(threshold, data, percentile_func, name='threshold'):
         raise TypeError('%s should be either a number '
                         'or a string finishing with a percent sign' % (name, ))
     return threshold
+
+
+def check_parameters_megatrawls_datasets(inputs, standards, name):
+    """ Checks given inputs against standards for megatrawls datasets parameters.
+
+    If parameters are valid, then no error message is raised else error message
+    will be raised.
+
+    Parameters
+    ----------
+    inputs: list of str
+        list to be checked.
+    standards: list of str
+        The given inputs will be checked against this given standards.
+    name: str
+        Used for precise naming in error message
+    """
+    message = ("Invalid {0} name is given: {1}. "
+               "Please choose either of them {2}")
+
+    if isinstance(inputs, _basestring):
```

Comment:
:confused: why don't you say `isinstance(inputs, collection.SequenceType)` or something like that if you want your inputs to be a list.

Also amend the error message to say "sequence" and show the type of the inputs.